docs/eventing/sources/apiserversource/reference.md

```diff
@@ -0,0 +1,199 @@
+| [`spec`][kubernetes-overview] | Specifies the configuration information for this ApiServerSource object. | Required |
+| `spec.mode` | EventMode controls the format of the event. Set to `Reference` to send a `dataref` event type for the resource being watched. Set to `Resource` to send the full resource lifecycle event. Defaults to `Reference`. | Optional |
+| [`spec.owner`](#owner-parameter) | ResourceOwner is an additional filter to only track resources that are owned by a specific resource type. If ResourceOwner matches Resources[n] then Resources[n] is allowed to pass the ResourceOwner filter. | Optional |
+| [`spec.resources`](#resources-parameter) | The resources that the source tracks so it can send related lifecycle events from the Kubernetes ApiServer. Includes an optional label selector to help filter. | Required |
+| `spec.serviceAccountName` | The name of the ServiceAccount to use to run this source. Defaults to `default` if not set. | Optional |
+| [`spec.sink`](#sink-parameter) | A reference to an object that resolves to a URI to use as the sink. | Required |
+| [`spec.ceOverrides`](#cloudevent-overrides) | Defines overrides to control the output format and modifications to the event sent to the sink. | Optional |
+
+
+### Owner parameter
+
+ResourceOwner is an additional filter to only track resources that are owned by
+a specific resource type. If ResourceOwner matches Resources[n] then Resources[n]
+is allowed to pass the ResourceOwner filter.
+
+An `owner` definition supports the following fields:
+
+| Field | Description | Required or optional |
+|-------|-------------|----------------------|
+| `apiVersion` | API version of the resource to watch. | Required |
+| [`kind`][kubernetes-kinds] | Kind of the resource to watch More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds. | Required |
+
+#### Example: Owner parameter
+
+```yaml
+apiVersion: sources.knative.dev/v1
+kind: ApiServerSource
+metadata:
+ name: <apiserversource>
+ namespace: <namespace>
+spec:
+  ...
+  owner:
+    apiVersion: apps/v1
+    kind: Deployment
+  ...
+```
+
+
+### Resources parameter
+
+The `resources` parameter specifies the resources that the source tracks so that
+it can send related lifecycle events from the Kubernetes ApiServer.
+The parameter includes an optional label selector to help filter.
+
+A `resources` definition supports the following fields:
+
+| Field | Description | Required or optional |
+|-------|-------------|----------------------|
+| `apiVersion` | API version of the resource to watch. | Required |
+| [`kind`][kubernetes-kinds] | Kind of the resource to watch. | Required |
+| [`selector`] | LabelSelector Filters this source to objects to those resources pass the label selector. More info: http://kubernetes.io/docs/concepts/overview/working-with-objects/labels/#label-selectors | Optional |
```

Comment:
This looks strange in the rendered page: https://deploy-preview-3812--dev-knative.netlify.app/development/eventing/sources/apiserversource/reference/#resources-parameter